### DISCUSSION TITLE
Update ollama and ollama webui when installed on Docker

### DISCUSSION THREAD

**Discussion by Gualty** [open-webui/open-webui#708] [Q&A]
Hi,
I see that there is a new image of ollama for docker and I want to update it.
What is the best way to update both ollama and webui?
I installed using the docker compose file reported in the installation guide

Thanks

**Comment by me9a6yte** [comment]
>What is the best way to update both ollama and webui?
this -> https://github.com/containrrr/watchtower

**Reply by encryptic12** [reply]
I had this running as recommended in the docs but it was not updating correctly so I have gone back to manually updating my Docker container(s). Maybe I was missing something. Any chance for some guidance on best practices? It has been a nuanced dance to get the open-webui/pipelines/ollama containers in rhythm and keep them that way.
